update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Dishant Khanna <dishant.khanna1807@gmail.com> [FOSSASIA]
+
+# Description:
+#   An update bash script for engelsystem
+
+# Add upstream 
+git remote add upstream http://www.github.com/fossasia/engelsystem.git
+
+# Pull the updates
+git pull upstream development
```

Comment:
I test on my local system, pulling the changes from my repository making the changes in the Version.txt and running the script file. [which includes pulling from my repository]